src/main/java/com/gmail/goosius/siegewar/settings/ConfigNodes.java

```diff
@@ -744,10 +740,10 @@ public enum ConfigNodes {
 			"#   (see user guide for full details)",
 			"#"),
 	PEACEFUL_CAPITALS_ENABLED(
-			"neutral_towns.capitals_allowed_neutrality",
-			"true",
+			"neutral_towns.nation_capital_can_be_neutral",

```

Comment:
This setting should have a migration added.

Reply:
- This lack of migration is deliberate, to force-change it for servers for their own protection.
- Because with the existing occupation system, allowing peaceful capitals is a minor preference. If subverted, the town just get occupied and the nation continues much as before.
- However with the updated occupation system, allowing peaceful capitals is a big deal. Because if a capital is subverted, that town is removed from the nation, with possibly severe consequences, including a random mayor from another town becoming king, or the nation being disbanded.